includes/abilities-api/class-wp-ability.php

```diff
@@ -266,6 +281,17 @@ public function get_meta(): array {
 		return $this->meta;
 	}
 
+	/**
+	 * Checks whether the ability should be shown in the REST API.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if the ability should be shown in the REST API, false otherwise.
+	 */
+	public function has_show_in_rest(): bool {
```

Comment:
I think the "has" is redundant in this case, as `$ability->show_in_rest()` is perfectly legible.
```suggestion
	public function show_in_rest(): bool {
```

Reply:
Good point.

Reply:
Addressed in[9f21064](https://github.com/WordPress/abilities-api/pull/103/commits/9f21064ca08f2ce50553f8fea8b0acd6948bd61c).